bundles/org.openhab.binding.amberelectric/src/main/java/org/openhab/binding/amberelectric/internal/AmberElectricHandler.java

```diff
@@ -89,8 +92,7 @@ public void initialize() {
         }
 
         webTargets = new AmberElectricWebTargets();
-        updateStatus(ThingStatus.UNKNOWN);
-        refreshInterval = config.refresh;
+        updateStatus(ThingStatus.ONLINE);
```

Comment:
This should not need to be changed, the `pollStatus` will set the thing to online when everything works as expected.
```suggestion
        updateStatus(ThingStatus.UNKNOWN);
```